src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -1,127 +0,0 @@
-package com.terraformation.backend.accelerator.api
```

Comment:
This class is renamed to ModulesController, and in that class, we added abilities to query by participants and cohorts in addition.